### DISCUSSION TITLE
Surface lacquer is ugly, how do we fix it

### DISCUSSION THREAD

**Discussion by Konclan** [BEEmod/BEE2-items#4205] [General]
This is a discussion thread on ideas on how to improve the visuals of surface lacquer in a way that looks both good in large scale and isn't too technically demanding.

**Comment by TeamSpen210** [comment]
It's tricky, we need something that implies gel can't coat it, indeed looks good over large areas, and still is transparent enough that you can see the colour of the underlying surface.

**Comment by Critfish** [comment]
Instead of applying an overlay to an existing texture, perhaps the surface lacquer should be given its own completely separate texture, one for black surfaces and one for white, with different variants for each style. It could probably have a more solid-colored, glazed ceramic look to it, similar to bathroom tiles. This'd have the extra benefit of making them more visible for users who have their shader quality setting set to low.

**Comment by TeamSpen210** [comment]
That could be done yeah, but we'd need to make a huge heap of textures - white, black, 2x2, 4x4, floor, ceiling - for all the styles.
